pkg/balancer-js/src/pool-weighted/encoder.ts

```diff
@@ -4,7 +4,8 @@ import { BigNumberish } from '@ethersproject/bignumber';
 export enum WeightedPoolJoinKind {
   INIT = 0,
   EXACT_TOKENS_IN_FOR_BPT_OUT,
-  TOKEN_IN_FOR_EXACT_BPT_OUT,
+  ONE_TOKEN_IN_FOR_EXACT_BPT_OUT,
+  TOKENS_IN_FOR_EXACT_BPT_OUT,
```

Comment:
I'm not a fan of the rename - ideally this would be as backwards compatible as possible. I'd just name the new one something different, like `ALL_TOKENS_IN...` or maybe even `PROPORTIONAL_TOKENS_IN_...`

Reply:
Done - didn't notice you'd already answered this question here.